gregoriotex/index.html

```diff
@@ -56,7 +56,7 @@
 </script>
 <h1>Gregorio<span class="tex">T<span class="epsilon">e</span>X</span></h1>
 
-<p>This page describes quickly the origin and the construction of Gregorio<span class="tex">T<span class="epsilon">e</span>X</span>. You can also take a look at the <a href="./introduction.html">Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> introduction page</a> or at the <a href="./details.html">Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> detailed description page</a>. If it is the first time you are dealing with <span class="tex">T<span class="epsilon">e</span>X</span>, I advise you to take a look at the <a href="./tex.html"><span class="tex">T<span class="epsilon">e</span>X</span> page</a>.</p>
+<p>This page describes quickly the origin and the construction of Gregorio<span class="tex">T<span class="epsilon">e</span>X</span>. You can also take a look at the <a href="./details.html">Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> introduction page</a> or at the <a href="./details.html">Gregorio<span class="tex">T<span class="epsilon">e</span>X</span> detailed description page</a>. If it is the first time you are dealing with <span class="tex">T<span class="epsilon">e</span>X</span>, I advise you to take a look at the <a href="./tex.html"><span class="tex">T<span class="epsilon">e</span>X</span> page</a>.</p>
```

Comment:
Right, but there is no GTeX introduction page. So I will just delete the first half of the sentence?